corehq/apps/geospatial/models.py

```diff
@@ -143,5 +143,5 @@ class GeoConfig(models.Model):
 
     domain = models.CharField(max_length=256)
     location_data_source = models.CharField(max_length=126, default=CUSTOM_USER_PROPERTY)
-    custom_user_property_name = models.CharField(max_length=256, default=GEO_POINT_CASE_PROPERTY)
+    user_location_property_name = models.CharField(max_length=256, default=GEO_POINT_CASE_PROPERTY)
```

Comment:
If I don't already have a PR open, or if my PR doesn't have any comments yet, I like to rebase when I change my mind, to keep the changeset clean. (And also to make people think I'm always right the first time. :joy: )